php/coresky/config.yaml

```
@@ -0,0 +1,14 @@
+framework:
+  website: coresky.net
+  version: 0.508
```

```suggestion
  version: 0.501
```

:information_source:  There will be an update PR on each `coresky` update (made by a bot)

---

Please use 0.508